pkg/api/common/versions.go

```diff
@@ -389,12 +389,12 @@ var AllKubernetesSupportedVersionsAzureStack = map[string]bool{
 	"1.22.7":  false,
 	"1.22.15": false,
 	"1.22.16": false,
-	"1.22.17": true,
 	"1.23.6":  false,
 	"1.23.12": false,
 	"1.23.13": false,
 	"1.23.14": false,
 	"1.23.15": true,
+	"1.24.7":  false,
```

Comment:
Let's keep the ASH tables in sync with `AllKubernetesSupportedVersions`

Reply:
Disabled1.22.17 in AllKubernetesSupportedVersions to keep in sync

Reply:
Disabled 1.20.15 and 1.21.14 as well